src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1753,6 +1753,69 @@ internal PointerTypeSymbol CreatePointerTypeSymbol(TypeSymbol elementType)
 
         #endregion
 
+        #region Operations
+
+        /// <summary>
+        /// Gets the low-level operation corresponding to the method's body.
+        /// </summary>
+        /// <param name="method">The method symbol.</param>
+        /// <param name="cancellationToken">An optional cancellation token.</param>
+        /// <returns>The low-level operation corresponding to the method's body.</returns>
+        protected override IOperation GetOperationCore(IMethodSymbol method, CancellationToken cancellationToken = default)
+        {
+            IOperation result = null;
+            var csmethod = method.EnsureCSharpSymbolOrNull<IMethodSymbol, MethodSymbol>(nameof(method));
+
+            if (csmethod != null)
```

Comment:
>if (csmethod != null) [](start = 12, length = 21)

We also need to make sure the method is actually part of the source module of this compilation.